coding/coding_tests/reader_writer_ops_test.cpp

```diff
@@ -119,3 +120,35 @@ UNIT_TEST(ReadWrite_POD)
 
   TEST(equal(src.begin(), src.end(), dest.begin()), ());
 }
+
+namespace
+{
+template <class T> void TestIntegral()
+{
+  std::vector<T> ethalon{ static_cast<T>(-1), 0, 1,
```

Comment:
etalon

Reply:
"ethalon" here.
Anyway, there are no Eng words like "etalon" or "ethalon", but Ethalon is used in string matching.